workshops/templates/workshops/workshop_staff.html

```diff
@@ -13,21 +13,25 @@
   {% if persons %}
     <table class="table table-striped">
       <tr>
-	<th>DC?</th>
-	<th>SWC?</th>
+        <th>instr. badges</th>
         <th>person</th>
         <th>taught</th>
+        <th>trainee?</th>
         <th>airport</th>
         <th>country</th>
         <th>lessons</th>
         <th>affiliation</th>
       </tr>
     {% for p in persons %}
       <tr>
-	<td class="text-center">{% if p.is_dc_instructor %}{% bootstrap_tag "DC" %}{% endif %}</td>
-	<td class="text-center">{% if p.is_swc_instructor %}{% bootstrap_tag "SWC" %}{% endif %}</td>
+        <td>{% for badge in p.badges.all %}
+        {% if badge in instructor_badges %}
+          {% bootstrap_tag badge.name|cut:"-instructor"|upper %}
+        {% endif %}
+        {% endfor %}</td>
```

Comment:
There's nothing wrong with this indentation :-) the removed code used wrong indentation (tabs).